daemons/fenced/fenced_commands.c

```diff
@@ -100,7 +100,7 @@ typedef struct async_command_s {
     char *device;
 
     GList *device_list;
-    GList *device_next;
+    GList *device_next; // Iterator for device_list
```

Comment:
that struct is local - so we could name it to something that tells what it is maybe device_iter_next.
anyway this device_next and next_device doesn't read easily.

Reply:
Good point, that did bug me too

Reply:
I'm thinking next_device_iter since next_device is the device corresponding to it

Reply:
found it appealing that everything here starts with device_ - device_next could be used as the block-variable.
on the other hand next_device is more like natural language. so either way fine with me.